server/views/pages/view/cases.njk

```diff
@@ -83,100 +87,197 @@
 {% if probationView %} {% set probationViewUrlQuery = '?view=probation' %} {% endif %}
 
 {% block content %}
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-three-quarters">
-            <h1 class="govuk-heading-xl">Select a person to view or print licences and supervision orders</h1>
+    {% if HardStopEnabled %}
```

Comment:
I'm finding this really difficult to read, maybe splitting out some macros would help?
```
 {% if HardStopEnabled %}
   { call hardstop macro }
else
   { call non-hardstop macro }
end
```